doc/doc.cabal

```diff
@@ -1,11 +1,11 @@
-name:                doc
-version:             0.1.0.0
-license:             BSD3
-license-file:        LICENSE
-author:              Julian K. Arni
-maintainer:          jkarni@gmail.com
-build-type:          Simple
-cabal-version:       >=1.10
+name:          doc
+version:       0.1.0.0
+license:       BSD3
+license-file:  LICENSE
+author:        Julian K. Arni
+maintainer:    jkarni@gmail.com
```

Comment:
@jkarni do you still want to be listed as maintainer, or shall it be maintained by current maintainers instead?

Reply:
I’m happy to be removed!

Reply:
🙂‍↕️

Reply:
Well done, GitHub, this is *exactly* the emoticon I sent 🥲